src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,19 +37,24 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={searchQuery}
+        onChange={e => setSearchQuery(e.target.value)}
```

Comment:
Move it to a separate function instead of using an anonymous function

Fix it everywhere it's possible